cli/cmd/asset_create.go

```diff
@@ -0,0 +1,143 @@
+// Copyright © 2023 Cisco Systems, Inc. and its affiliates.
+// All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/spf13/cobra"
+
+	"github.com/openclarity/vmclarity/api/models"
+	"github.com/openclarity/vmclarity/shared/pkg/backendclient"
+)
+
+// assetCreateCmd represents the standalone command.
+var assetCreateCmd = &cobra.Command{
+	Use:   "asset-create",
+	Short: "Create asset",
+	Long:  `It creates asset. It's useful in the CI/CD mode without VMClarity orchestration`,
+	Run: func(cmd *cobra.Command, args []string) {
+		logger.Infof("Creating asset...")
+		filename, err := cmd.Flags().GetString("from-json-file")
+		if err != nil {
+			logger.Fatalf("Unable to get asset json file name: %v", err)
+		}
+		server, err := cmd.Flags().GetString("server")
+		if err != nil {
+			logger.Fatalf("Unable to get VMClarity server address: %v", err)
+		}
+
+		assetType, err := getAssetFromJSONFile(filename)
+		if err != nil {
+			logger.Fatalf("Failed to get asset from json file: %v", err)
+		}
+
+		_, err = assetType.ValueByDiscriminator()
+		if err != nil {
+			logger.Fatalf("Failed to determine asset type: %v", err)
+		}
+
+		assetID, err := createAsset(context.TODO(), assetType, server)
+		if err != nil {
+			logger.Fatalf("Failed to create asset: %v", err)
+		}
+		fmt.Println(assetID)
```

Comment:
I would rather than this returned the json by default, as you would get in a curl, and then we add an option to filter it down to a selected field, for example we can use https://pkg.go.dev/k8s.io/client-go/util/jsonpath to implement jsonpath support like `--jsonpath {.id}`